src/_cffi_src/openssl/cryptography.py

```diff
@@ -68,6 +71,10 @@
 #else
 #define CRYPTOGRAPHY_NEEDS_OSRANDOM_ENGINE 0
 #endif
+/* Ed25519 support is available from OpenSSL 1.1.1b and LibreSSL 3.7.0. */
+#define CRYPTOGRAPHY_HAS_WORKING_ED25519 \
+    (!CRYPTOGRAPHY_OPENSSL_LESS_THAN_111B || \
+    !CRYPTOGRAPHY_LIBRESSL_LESS_THAN_370)
```

Comment:
```suggestion
    (CRYPTOGRAPHY_IS_LIBRESSL && !CRYPTOGRAPHY_LIBRESSL_LESS_THAN_370))
```

Reply:
Yes, just noticed by the test failure. Will fix.